Corona/Model/Change.swift

```diff
@@ -36,13 +36,13 @@ extension Change {
 		lastDeaths == 0 ? 0 :(Double(currentDeaths) / Double(lastDeaths) - 1) * 100
 	}
 
-	public var newConfirmedString: String { "+\(newConfirmed.groupingFormatted)" }
+    	public var newConfirmedString: String { newConfirmed > 0 ? "+\(newConfirmed.groupingFormatted)" : "-" }
 	public var newRecoveredString: String { currentRecovered == 0 ? "-" : "+\(newRecovered.groupingFormatted)" }
-    public var newDeathsString: String { (lastDeaths < currentDeaths) ? "+\(newDeaths.groupingFormatted)" : "\(newDeaths.groupingFormatted)" }
+    	public var newDeathsString: String { newDeaths > 0 ? "+\(newDeaths.groupingFormatted)" : "-" }
 
-	public var confirmedGrowthString: String { "↑\(confirmedGrowthPercent.kmFormatted)%" }
+    	public var confirmedGrowthString: String { newConfirmed > 0 ? "↑\(confirmedGrowthPercent.kmFormatted)%" : "-" }
```

Comment:
Actually, now that i think about it should this be `currentConfirmed`? What do you think @mhdhejazi ?

Reply:
Well, the problem happens when `currentConfirmed` is less than `lastConfirmed` not zero. So, checking for  `newConfirmed > 0` is the right thing here.